### DISCUSSION TITLE
Advanced Stock Management

### DISCUSSION THREAD

**Discussion by Dreimus** [PrestaShop/PrestaShop#33941] [Q&A]
Hello there !

I've seen in the Prestashop Core Monthly that Advanced Stock Management code is being cleared out from Prestashop 9.
I understand that this is probably legacy code that doesn't need to be kept. 

But a multiple warehouse management would be a great feature for Prestashop, as many merchant use it. 

Is is something planned on the roadmap about this ?

**Comment by MatShir** [comment]
Hi Paul, 
Very pleased to hear that you are reading Core Monthly 🤩 

To get back on the subject. There are no specific plans for the Advanced Stock, but we can utilize this discussion to gather feedback. What essential features are lacking in the core system for your agency to manage multiple storage locations? It seems that many users rely on their ERP or logistics provider to handle stock management. Could you please provide more information on this topic?

**Reply by Dreimus** [reply]
The main subject is the Warehouse Management, there are many merchants that don't use ERP for synchronisation or want to use Prestashop as a "mirror" stock manager. Some of them rely on multiple warehouses. The quantity of each product can be different depending on warehouses. It can be interesting to allow the carrier to create stock movements on a warehouse quantity based on the delivery zone and the customer address.

The shipping costs and delivery times can be differents depending on the warehouse proximity with the customer.

**Reply by Dreimus** [reply]
I think this can be a key feature leading merchants to chose another solution instead of Prestashop as it is not natively supported.
